Dockerfile

```diff
@@ -1,11 +1,13 @@
 # Keep hub.Dockerfile aligned to this file as far as possible
 ARG JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
+ARG APP_INSIGHTS_AGENT_VERSION=2.3.1-SNAPSHOT
 
-FROM hmcts/cnp-java-base:openjdk-8u191-jre-alpine3.9-2.0.1
+FROM hmctspublic.azurecr.io/base/java:openjdk-8-distroless-1.0
 LABEL maintainer="https://github.com/hmcts/ccd-user-profile-api"
 
 COPY build/libs/user-profile.jar  /opt/app/
 COPY lib/AI-Agent.xml /opt/app
+COPY lib/applicationinsights-agent-2.3.1-SNAPSHOT.jar lib/AI-Agent.xml /opt/app/
 
 HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" wget -q --spider http://localhost:4553/status/health || exit 1
```

Comment:
according to TimJ that new image doesn't have bash so this health check might fail:
please check here:
https://github.com/hmcts/ccd-data-store-api/pull/535#discussion_r316655268

Reply:
perhaps before removing it we might see if it works. It'll be good to keep it

Reply:
removed HEALTHCHECK